plugins/ucrm-esignature-platform/src/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "1",
+    "information": {
+        "name": "contract-plugin",
+        "displayName": "eSignature Platform - 07INTERNET.RO",
+        "description": "You can visualize and sign your contract directly on the platform.",
+        "url": "https://github.com/matei1337/ucrm-esignature-platform",
+        "version": "1.0.0",
+        "ucrmVersionCompliancy": {
```

Comment:
UCRM v2 is no longer officially supported, there is no need to include `ucrmVersionCompliancy` anymore